Blocks/Persistence/file.h

```diff
@@ -43,6 +43,8 @@ SOFTWARE.
 #include "../../Bricks/time/chrono.h"
 #include "../../Bricks/sync/scope_owned.h"
 
+#include "../../Bricks/util/atomic_that_works.h"  // TODO(dkorolev): Remove once GCC 5 is mainstream.
```

Comment:
This doesn't quite nail it, unfortunately. See https://github.com/c5t/Current/pull/394.